mutiny-core/src/lib.rs

```diff
@@ -440,6 +439,7 @@ impl<S: MutinyStorage> MutinyWallet<S> {
     pub(crate) async fn start_nostr_wallet_connect(&self, from_node: PublicKey) {
         let nostr = self.nostr.clone();
         let nm = self.node_manager.clone();
```

Comment:
i think we're only getting the node manager for the logger now, could just replace with self.logger

Reply:
Glad you caught this. It's more involved because `node` is being retrieved and called upon actually. Not everywhere was it using the `LnPayer` trait to find the status of a payment. But I should be able to get this tonight.